subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -1026,6 +1029,18 @@ static int lwm2m_send_message(struct lwm2m_message *msg)
 	return 0;
 }
 
+int lwm2m_dump_notifications(struct lwm2m_ctx *client_ctx)
```

Comment:
I think `dump` might be a bit confusing in this case, as it might suggest we're dropping the stotred notifications, not sending them. Perhaps `resend` would be more appropriate.